### ISSUE TITLE
Recover from Rsync error

### ISSUE THREAD

**Issue by rmahoney-bl** [smerrill/vagrant-gatling-rsync#16]
For a variety of reasons, a transient error can cause rsync to temporarily fail.  It would be nice if the plugin could detect the failure better, and retry indefinitely

**Comment by nfloersch** [comment]
Agreed. This happens more on my office machine than my home machine... working fine for a while then bam it falls over. Still it is more reliable than the built in vagrant rsync-auto which frequently fails to sync after just a couple of rounds.
